authors/apps/core/utils.py

```diff
@@ -6,13 +6,15 @@ class Utilities:
     """Utility class that contains helper function"""
 
     @staticmethod
-    def email_renderer(data):
+    def send_email(data,intent):
```

Comment:
@CryceTruly  can you make intent a kwarg with default None? This can reduce on the bugs of where method send_email is used in the codebase

Reply:
okay @marthamareal, thanks for that.